reactive-banana/src/Reactive/Banana/Prim/Compile.hs

```diff
@@ -20,14 +21,14 @@ import           Reactive.Banana.Prim.Types
 -- | Change a 'Network' of pulses and latches by
 -- executing a 'BuildIO' action.
 compile :: BuildIO a -> Network -> IO (a, Network)
-compile m (Network time1 outputs1 theAlwaysP) = do
-    (a, topology, os) <- runBuildIO (time1, theAlwaysP) m
+compile m Network {nTime, nOutputs, nAlwaysP} = do
```

Comment:
```suggestion
compile m Network{nTime, nOutputs, nAlwaysP} = do
```

It's probably a matter of taste, but I like to leave no space between the constructor and the curly braces, lest they can be confused for two separate arguments.